src/commands/toolchain.rs

```diff
@@ -42,6 +50,20 @@ pub struct UninstallCommand {
 #[derive(Debug, Parser)]
 pub struct ListRevisionsCommand {}
 
+#[derive(Debug, Deserialize, Parser)]
+pub struct ExportCommand {
+    /// Toolchain to export, [possible values: latest, beta-1, beta-2, beta-3, beta-4, nightly].
+    /// The default toolchain will be exported if name isn't specified
+    pub name: Option<String>,
+    /// Channel to export, [possible values: latest-YYYY-MM-DD, nightly-YYYY-MM-DD, beta-1, beta-2, beta-3, beta-4].
+    #[clap(value_parser = channel_allowed)]
```

Comment:
clap will add the possible values to the help message for you:

```suggestion
    /// Channel to export, [possible values: latest-YYYY-MM-DD, nightly-YYYY-MM-DD, beta-1, beta-2, beta-3, beta-4].
    #[clap(value_parser = clap::builder::PossibleValuesParser::new(["latest-YYYY-MM-DD", "nightly-YYYY-MM-DD", "beta-1", "beta-2", "beta-3", "beta-4"]))]
```

Reply:
@sdankel  Hi sir, I think using `clap::builder::PossibleValuesParser` would cause unexpected behaviour. Will clap parse "YYYY-MM-DD" correctly? I tried to provide `latest-2023-12-13` but failed.

![image](https://github.com/FuelLabs/fuelup/assets/25278203/c9e24d04-1a83-4209-b8b8-8175c307514a)